tests/test_cart.py

```diff
@@ -1,15 +1,231 @@
 from __future__ import unicode_literals
 
 import json
+from uuid import uuid4
 
 import pytest
 from babeldjango.templatetags.babel import currencyfmt
 from django.core.exceptions import ObjectDoesNotExist
+from django.http import HttpRequest
 from mock import MagicMock, Mock
+from satchless.item import InsufficientStock
 
-from saleor.cart import decorators, forms, utils
+from saleor.cart import forms, utils
+from saleor.cart.decorators import (find_and_assign_cart,
+                                    get_anonymous_cart_from_token,
+                                    get_cart_from_request,
+                                    get_or_create_anonymous_cart_from_token,
+                                    get_or_create_cart_from_request,
+                                    get_or_create_user_cart, get_user_cart)
 from saleor.cart.models import Cart
-from satchless.item import InsufficientStock
+
+
+@pytest.fixture()
+def test_user(django_user_model):
```

Comment:
py.test has builtin `admin_user` fixture. 
Also there are already existing fixtures in saleor: https://github.com/mirumee/saleor/blob/master/tests/conftest.py#L26
https://github.com/mirumee/saleor/blob/master/tests/conftest.py#L53

You can simply use them by using their names as test function parameters (fixtures from conftest.py are imported automatically)